addons/account/tests/test_account_journal_dashboard.py

```diff
@@ -276,7 +276,7 @@ def test_gap_in_sequence_warning(self):
         self.assertFalse(journal._query_has_sequence_holes())  # no gap, no gap warning, and draft moves shouldn't trigger the warning
 
         moves[2:4].button_draft()
-        self.assertFalse(journal._query_has_sequence_holes())  # no gap (with draft moves using sequence numbers), no gap warning
+        self.assertTrue(journal._query_has_sequence_holes())  # gap due to draft moves using sequence numbers, gap warning
```

Comment:
So as soon as there is a move reset to draft with a sequence, we show it on the dashboard?
Seems weird.

Reply:
Yes, except if it's the last move